From 2acfa0818851e9a3ac300d59acf2cdcfd1899559 Mon Sep 17 00:00:00 2001
From: flintman <flintman@flintmancomputers.com>
Date: Thu, 7 Nov 2013 19:50:06 -0500
Subject: [PATCH] build: Add SKIP_SET_METADATA flag

On some of the older devices the recovery doesn't support
metadata when setting permissions. This allows to use
set_perm vs set_metadata.
Enable this behavior by setting SKIP_SET_METADATA := true

Change-Id: Ib6cf97b5ca28cee8a87975231f3a61b156e77d29
---
 core/Makefile | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/core/Makefile b/core/Makefile
index aa7b88d..ce30a34 100644
--- a/core/Makefile
+++ b/core/Makefile
@@ -1376,7 +1376,9 @@ ifdef PRODUCT_EXTRA_RECOVERY_KEYS
 	$(hide) echo "extra_recovery_keys=$(PRODUCT_EXTRA_RECOVERY_KEYS)" >> $(zip_root)/META/misc_info.txt
 endif
 	$(hide) echo "mkbootimg_args=$(BOARD_MKBOOTIMG_ARGS)" >> $(zip_root)/META/misc_info.txt
+ifneq ($(SKIP_SET_METADATA),true)
 	$(hide) echo "use_set_metadata=1" >> $(zip_root)/META/misc_info.txt
+endif
 	$(hide) echo "update_rename_support=1" >> $(zip_root)/META/misc_info.txt
 	$(call generate-userimage-prop-dictionary, $(zip_root)/META/misc_info.txt)
 	@# Zip everything up, preserving symlinks
-- 
1.8.3.2


